helm/opendistro-es/files/configurer/configurer.sh

```diff
@@ -180,14 +180,99 @@ create_role_definitions() {
 
   create_role "kubernetes-log-reader" '{ "index_permissions": [{ "index_patterns": ["kubernetes*", "kubeaudit*"], "allowed_actions": ["read"]}]}'
   create_role "backup-exporter" '{"cluster_permissions": ["cluster:monitor/state","cluster:monitor/health"], "index_permissions": [{"index_patterns": ["*"], "allowed_actions": ["monitor"]}, {"index_patterns": ["kubernetes*", "kubeaudit*"], "allowed_actions": ["read"]}]}'
+  #create roles for different users
+  roles="{{ toJson .Values.configurer.roles }}"
+  for row in $(echo "$roles"  | jq -r '.[] | @base64');
+  do
+      _jq()
+      {
+        echo ${row} | base64 --decode | jq -r ${1}
+      }
+      roles_name= $(_jq '.roles_name')
+      roles_definition=  $(_jq '.roles_definition')
+      create_role $roles_name $roles_name
+  done
 }
+$ curl -XPUT https://localhost:9200/_opendistro/_security/api/roles/my-curl-role -u admin:admin -k -H 'Content-Type: application/json' -d '{"cluster_permissions":["cluster_composite_ops","indices_monitor"],"index_permissions":[{"index_patterns":["movies*"],"dls":"","fls":[],"masked_fields":[],"allowed_actions":["read"]}],"tenant_permissions":[{"tenant_patterns":["human_resources"],"allowed_actions":["kibana_all_read"]}]}'
```

Comment:
Should this line be here? :slightly_smiling_face:

Reply:
> Should this line be here?

That is right. That shouldn't be there!